tomcat/src/main/java/support/StreamUtils.java

```diff
@@ -0,0 +1,29 @@
+package support;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+public class StreamUtils {
+
+    private StreamUtils() {
+    }
+
+    public static String readAllLines(InputStream inputStream) {
+        final StringBuffer stringBuffer = new StringBuffer();
+        try {
+            final BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream, "UTF-8"));
+            while (true) {
+                if (!bufferedReader.ready()) {
+                    break;
+                }
+                final String line = bufferedReader.readLine();
+                stringBuffer.append(line);
+            }
```

Comment:
```suggestion
            while (bufferedReader.ready()) {
                final String line = bufferedReader.readLine();
                stringBuffer.append(line);
            }
```
이렇게 해도 같지 않나요? 무슨 이유 때문에 위와 같은 형식으로 작성하신건지 궁금해요! 🤔

Reply:
아, 현재는 저렇게 수정되었습니다 !!.. ㅎㅎ suggestion 감사합니다 ㅎ